gtk-3.0/gtk-widgets.css

```diff
@@ -2880,6 +2880,31 @@ button.destructive-action:active:hover,
     border-color: shade (@error_color, 0.85);
 }
 
+button.destructive-action:focus,
+.titlebar button.destructive-action:focus,
+.destructive-action.button:focus,
+.titlebar .destructive-action.button:focus {
+    background-image:
+        linear-gradient(
+            to bottom,
+            shade (
+                @error_color,
+                1.4
+            ),
+            shade (
+                @error_color,
+                1.1
+            )
+        );
+    box-shadow:
+        inset 0 1px 0 0 alpha (#fff, 0.3),
+        inset 0 -1px 0 0 alpha (#fff, 0.2),
+        0 1px 0 0 alpha (#fff, 0.25),
```

Comment:
This is a little aggressive in the dark stylesheet. Maybe use `@bg_highlight_color` instead of `#fff` on this line?